src/test/java/com/coveo/spillway/SpillwayTest.java

```diff
@@ -384,4 +394,74 @@ public void triggersAreIgnoreIfTheStorageReturnsAnIncoherentResponse() {
 
     verify(callback, never()).trigger(any(LimitDefinition.class), any());
   }
+
+  @Test
+  public void canAddCapacityLimitOverride() {
+    LimitOverride override = LimitOverrideBuilder.of(JOHN).to(A_CAPACITY).per(A_DURATION).build();
```

Comment:
I find the idea of another property extractor in the override interesting. That would give a nice boost to the possible use case.
@francistb what do you think?